gpsoauth/__init__.py

```diff
@@ -25,7 +26,7 @@
 ANDROID_KEY_7_3_29 = google.key_from_b64(B64_KEY_7_3_29)
 
 AUTH_URL = "https://android.clients.google.com/auth"
-USER_AGENT = "gpsoauth/" + __version__
+USER_AGENT = "GoogleAuth/1.4"
```

Comment:
I liked being transparent about not being a real google client, but if the user agent is required then I'm ok with using it.

Reply:
i've just went ahead and checked which of the changes is actually required, and it seems the only crucial change is the SSLcontext. the user agent can stay as gpsoauth. as for the signatures and droidguard result, they aren't actually required but may be good to include as a sort of future proofing.

Reply:
Great. Let's just go with the minimal change for now, and I'll add in the other changes if they become needed later on.

Reply:
alright, i've reverted the unnecessary changes

Reply:
Interesting, this revision actually didn't work for my test account!

Sorry for the back and forth -- I'm happy to go with original set of changes since I'm too lazy to figure out exactly which ones are required 😛

Reply:
oh that's weird, just tried here and it works fine :thinking: 
fine by me either way, just wanted to get other people back up and running again :)

Reply:
I hear ya! I appreciate it!